types/http-errors/http-errors-tests.ts

```diff
@@ -1,86 +1,74 @@
-
 import * as createError from 'http-errors';
 import * as express from 'express';
 
-var app = express();
-
-declare global {
-    namespace Express {
-        export interface Request {
-            user?: any
-        }
-    }
-}
+const app = express();
 
-app.use(function (req, res, next) {
-    if (!req.user) return next(createError(401, 'Please login to view this page.'));
+app.use((req, res, next) => {
+    if (!req) return next(createError('Please login to view this page.', 401));
     next();
 });
 
 /* Examples taken from https://github.com/jshttp/http-errors/blob/1.3.1/test/test.js */
```

Comment:
Old http-errors-tests.ts is https://github.com/jshttp/http-errors/blob/1.3.1/test/test.js
The new version of http-errors-tests.ts then should be https://github.com/jshttp/http-errors/blob/1.6.2/test/test.js